pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -128,6 +139,11 @@ type Framework interface {
 	// plugins returns an error, it does not continue running the remaining ones and
 	// returns the error. In such case, pod will not be scheduled.
 	RunReservePlugins(pc *PluginContext, pod *v1.Pod, nodeName string) *Status
+
+	// RunUnreservePlugins runs the set of configured unreserve plugins. If any of these
+	// plugins returns an error, it does not continue running the remaining ones and
+	// returns the error.
+	RunUnreservePlugins(pc *PluginContext, pod *v1.Pod, nodeName string) *Status
```

Comment:
If we decide to not have any return value for `Unreserve` plugins, we won't need any return value here either.

Reply:
Addressed.